v2/openapi/fb-provider-crosstransfer-api.yaml

```diff
@@ -0,0 +1,304 @@
+        '400':
+          $ref: '#/x-responses/BadRequestResponse'
+        '401':
+          $ref: '#/x-responses/UnauthorizedResponse'
+        '4XX':
+          $ref: '#/x-responses/ErrorResponse'
+        '5XX':
+          $ref: '#/x-responses/ErrorResponse'
+
+  /crosstransfer/bridging:
+    post:
+      operationId: createCrossTransferBridging
+      summary: >-
+        Creates deposit-only crypto account as a source, configures trigger and stores 
+        payment instruction details about destination crypto address. As a result, the user will be
+        able to deposit crypto to the source account and receive exchanged value to destination address 
+        in selected cryptocurrency.
+      tags: [ crosstransfers ]
+      parameters:
+        - $ref: '#/x-header-params/X-FBAPI-KEY'
+        - $ref: '#/x-header-params/X-FBAPI-NONCE'
+        - $ref: '#/x-header-params/X-FBAPI-SIGNATURE'
+        - $ref: '#/x-header-params/X-FBAPI-TIMESTAMP'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateBridgingRequest'
+      responses:
+        '200':
+          description: Sub-account created.
+          content:
+            application/json:
+              schema:
+                $ref: '#/x-schemas/AccountId'
+        '400':
+          $ref: '#/x-responses/BadRequestResponse'
+        '401':
+          $ref: '#/x-responses/UnauthorizedResponse'
+        '4XX':
+          $ref: '#/x-responses/ErrorResponse'
+        '5XX':
+          $ref: '#/x-responses/ErrorResponse'
+
+components:
+  schemas:
+    CreateBridgingRequest:
+      type: object
+      required: [ idempotencyKey, customerId, source, destination, ]
+      properties:
+        idempotencyKey:
+          type: string
+          example: '76a33ed9-8aa2-4781-93f8-d2369c0c2b14'
+        customerId:
+          type: string
+          example: '76a33ed9-8aa2-4781-93f8-d2369c0c2b14'
+        source:
+          $ref: '#/components/schemas/BridgingSourceParameters'
+        destination:
+          $ref: '#/components/schemas/BridgingDestination'
+    BridgingDestination:
+      type: object
+      required:
+        - chain
+        - currency
+        - address
+      properties:
+        chain:
+          $ref: '#/x-schemas/Blockchain'
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        address:
+          description: Address of the destination account.
+          type: string
+        memo:
+          description: Memo field for the destination address. Required for some blockchains.
+          type: string
+    BridgingSourceParameters:
+      type: object
+      required: [ currency ]
+      properties:
+        chain:
+          $ref: '#/x-schemas/Blockchain'
+        currency:
+          $ref: '#/x-schemas/NativeCryptocurrency'
+        developerFee:
+          $ref: './fb-provider-api.yaml#/components/schemas/PositiveAmount'
```

Comment:
PublicBlockchainAddress object already defines a standard way to address blockchains. You can use all-of to add `developerFee`. Is the fee in the same currency as the source? Is it an absolute amount, a rate, percent? Is it a rational decimal number? I'd state it all explicitly in the name, if possible, and in the description of the object.